content/en/docs/Concepts/olm-architecture/operator-catalog/creating-an-update-graph.md

```diff
@@ -94,7 +94,7 @@ entries:
 
 The entry specifying the `skipRange` will be presented as a direct (one hop) update to
 any version from that package within that range. The versions in this range do not need to be in
-the index in order for bundle addition to be successful. We recommend avoiding using unbound ranges
```

Comment:
nit/question: what's the motivation for making "Catalog" a proper noun with a capitalized 'C' everywhere?

Reply:
That is a good point. `Catalog` is now catalog everywhere.